2009-02-13  Michael Tautschnig  <mt@debian.org>

	* bin/fai-chboot: Make -k work again, even if -i or -o is not used
		(closes: #495535)
	* man/fai-chboot.8: Don't claim -i does the same as -k ..., just give a few
		hints about the options set by default
Index: trunk/bin/fai-chboot
===================================================================
--- trunk.orig/bin/fai-chboot
+++ trunk/bin/fai-chboot	
@@ -443,7 +443,10 @@
 } else {
   $kernelname = shift;
   $kernelname = "kernel $kernelname";
-#  $rootfs = shift or die "No rootfs specified.\n";
+  $initrd     = '';
+  $rootfs     = '';
+  $bootprot = '';
+  $flags    = '';
 }
 
 $opt_F and $opt_f="verbose,sshd,createvt";
Index: trunk/man/fai-chboot.8
===================================================================
--- trunk.orig/man/fai-chboot.8
+++ trunk/man/fai-chboot.8	
@@ -79,7 +79,7 @@
 Show simple help and version.
 .TP
 .B \-i
-Set parameters for booting the FAI install kernel. Same as -k "ip=dhcp" vmlinuz-install /dev/nfs. This does not set FAI_ACTION.
+Set parameters for booting the FAI install kernel (proper initrd, boot from nfs). This does not set FAI_ACTION.
 .TP
 .B \-I
 Same as -i but also sets FAI_ACTION=install. So a fully automatic
